webservices/tasks/__init__.py

```diff
@@ -45,8 +46,17 @@ def redis_url():
         'webservices.tasks.legal_docs',
     ),
     CELERYBEAT_SCHEDULE=schedule,
+    acks_late=True,
 )
 
+app.conf.ONCE = {
+    'backend': 'celery_once.backends.Redis',
+    'settings': {
+        'url': redis_url(),
+        'default_timeout': 60 * 60
+    }
+}
```

Comment:
Just to make sure I understand correctly, this is different from the `app.conf.update` call above, right?  I'm assuming this is for Celery Once specifically, but it also appears that the original update above is making referencing to Celery Once defaults, which is part of what's confusing me.